### PR TITLE
Added recipe for intervals

### PR DESCRIPTION
**Pull request by pmlandwehr**
Added recipe for `intervals` and `infinity`, which it depends on.

### CODE REVIEW

File: recipes/infinity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
```
Comment (isuruf) [inline recipes/infinity/meta.yaml]: Do you want to try [`noarch: python`](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/meta.rst#building-noarch-packages)?

Comment (pmlandwehr) [reply recipes/infinity/meta.yaml]: I can, sure - should I also drop out the `script:` block below?

Comment (isuruf) [reply recipes/infinity/meta.yaml]: `script` block still needs to be there

**Comment by isuruf** [comment]
Thanks

**Pull request merged**